Sources/Util/Util.swift

```diff
@@ -22,6 +22,17 @@ public func isAnImage(_ file: String) -> Bool {
            file.hasSuffix("png")
 }
 
+public func getRange(of rawRange: String, length contentLength: Int) -> Range<Int> {
+  let splitRange = rawRange.components(separatedBy: "-")
+
+  let rangeEnd = Int(splitRange.last!) ?? contentLength - 1
```

Comment:
I think you should push yourself by treating all force-unwrapping as a design smell.